app/src/main/java/org/mifos/mobile/ui/fragments/ThirdPartyTransferFragment.java

```diff
@@ -39,6 +26,20 @@
 import org.mifos.mobile.utils.Toaster;
 import org.mifos.mobile.utils.Utils;
 
+import android.os.Bundle;
```

Comment:
Why remove the import lines and  add these lines again here?
Please update the PR , everything else seems fine to me :)

Reply:
I didnt get this?

Reply:
You just changed the placement of the import statements, this is not necessary